Creiwry/irl-meeting-app
# Add interest tag dropdown

### Ticket Link
(Found on Jira in your ticket description)
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-18

### User Story
(Linked to on Jira in your ticket description)
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-18

### What does this PR do?
(Your answer here.) 
This PR addresses the BACK only. A separate PR for the front, as well as for the back to edit and update sexual orientation, gender etc will come later.
- Adds interests array to seed
- Adds backend functionality to registrations#edit for the following:
   - adding and updating profile picture
   - adding and updating user_interests to profile
- adds current_user view to user profile that doesn't display interactions options and links to and from edit profile page

### Where should the reviewer start?
(Your answer here.)
review /users/edit and own profile page once signed in

### How should this be manually tested?
(Your answer here.)
- Check that you can click multiple interests in the dropdown and update your profile (they will display on the view of your profile from the link at the top of edit page)
- All rspec tests passed

### Screenshots (if appropriate)
(Your answer here.
![Screen Shot 2023-09-13 at 22 07 08](https://github.com/Creiwry/irl-meeting-app/assets/137942
![Screen Shot 2023-09-13 at 22 06 45](https://github.com/Creiwry/irl-meeting-app/assets/137942686/da6c379e-6924-431b-bfdb-d92c2063b0f4)
686/3
![Screen Shot 2023-09-13 at 22 07 17](https://github.com/Creiwry/irl-meeting-app/assets/137942686/a056b339-a401-4f8e-a9c4-87a6234792e1)
81be0ff-b16b-4365-b0ee-d5597a4f4550)
)

### Are there any breaking changes?
No
